packages/evo-marko/src/tags/evo-accordion/index.marko

```diff
@@ -0,0 +1,67 @@
+import type { Input as DetailsInput } from "<evo-details>";
+export interface Input extends Omit<Marko.HTML.UL, `on${string}`> {
```

Comment:
We will need to add a generic here because that way the type will be inferred automatically in the parent component
Also remove the `Omit`, its no longer needed

So just do this

```
export interface Input<Open extends number | number[] | undefined> extends Marko.HTML.UL {
   // other defitnitions
   open: Open;
   openChange: (open: Open) => void;
}
```